### PR TITLE
Fix #2546 Update CrowdAI status

### LINKED ISSUE #2546: [docs, urgent] Update CrowdAI status
The comparison table in the readme shows crowdAI, but according to this [blog post](https://www.crowdai.org/blogs/7), crowdAI has been shut down and replaced with a new closed-source platform AIcrowd.
Updates:
1. Change name CrowdAI -> AICrowd
2. Change open-source Yes -> No

### PR DESCRIPTION
**Pull request by apoorvkhare07**
Updated Readme.md to fix #2546. 
changed CrowdAI to AICrowd and it's status from open-source to closed-source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,12 +29,12 @@ In recent years, it has become increasingly difficult to compare an algorithm so
 - **Faster evaluation**: We warm-up the worker nodes at start-up by importing the challenge code and pre-loading the dataset in memory. We also split the dataset into small chunks that are simultaneously evaluated on multiple cores. These simple tricks result in faster evaluation and reduces the evaluation time by an order of magnitude in some cases.
 
 ## Platform Comparison
-|          Features          |          OpenML          |         TopCoder         |          Kaggle          |         CrowdAI          |          ParlAI          |         Codalab          |       EvalAI       |
+|          Features          |          OpenML          |         TopCoder         |          Kaggle          |         AICrowd          |          ParlAI          |         Codalab          |       EvalAI       |
```
Comment (nikochiko) [inline README.md]: AICrowd -> AIcrowd 
(lowercase 'c')

Comment (apoorvkhare07) [reply README.md]: should I make another commit incorporating this change?

Comment (nikochiko) [reply README.md]: Yes

**Comment by nikochiko** [comment]
@apoorvkhare07 LGTM :+1: Thanks for the contribution! :)

**Comment by nikochiko** [comment]
@apoorvkhare07 Extremely sorry for the mess up. But AIcrowd is still open-source. 
I misread the blog post. Can you please make the change?
Sorry again.

**Pull request closed without merging**